repotracker/wrappers.go

```diff
@@ -47,6 +54,30 @@ func CollectRevisionsForProject(conf *evergreen.Settings, project model.ProjectR
 	return nil
 }
 
+func ActivateBuildsForProject(conf *evergreen.Settings, project model.ProjectRef) error {
+	if !project.Enabled {
+		return errors.Wrap(errProjectDisabled, project.String())
+	}
+
+	tracker, err := getTracker(conf, project)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	if err = tracker.activationForProject(project.Identifier); err != nil {
```

Comment:
Why is this both in ActivateBuildsForProject and FetchRevisions?

Reply:
This just sees if batchtime has expired and activates any needed tasks in the most recent builds. It's safe to run in both places, and FetchRevisions as such only really runs in the legacy repotracker.